src/Compilers/CSharp/Portable/Syntax/ConstructorDeclarationSyntax.cs

```diff
@@ -39,7 +39,7 @@ public partial class SyntaxFactory
             SyntaxTokenList modifiers,
             SyntaxToken identifier,
             ParameterListSyntax parameterList,
-            ConstructorInitializerSyntax initializer,
+            ConstructorInitializerSyntax? initializer,
```

Comment:
part of me wonders if we can run some check to ensure that nullability of these kinds of factory methods remains synchronized with the nullability in Syntax.xml.